addons/compat_ace_fire/XEH_preInit.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+ADDON = false;
+
+GVAR(aceMedicalLoaded) = isClass(configFile >> "CfgPatches" >> "ace_medical_engine");
```

Comment:
can also exit out early if ace is loaded, as that is already handled by ace itself